components/script/dom/element.rs

```diff
@@ -762,7 +762,9 @@ impl<'dom> LayoutElementHelpers<'dom> for LayoutDom<'dom, Element> {
                 shared_lock,
                 PropertyDeclaration::FontFamily(font_family::SpecifiedValue::Values(
                     computed::font::FontFamilyList {
-                        list: HTMLFontElement::parse_face_attribute(font_face).into_boxed_slice(),
+                        list: ArcSlice::from_iter(
+                            HTMLFontElement::parse_face_attribute(font_face).into_iter(),
```

Comment:
This change appears to be required due to https://github.com/servo/stylo/commit/33c26645a906afbe35821ba567c7c92d96a3ba14, which has not been pulled into Servo since merging into stylo.